media/CircleEditor/index.js

```diff
@@ -46,7 +46,8 @@
 
 var host = {};
 
-const vscode = acquireVsCodeApi();
+var vscode = acquireVsCodeApi();
```

Comment:
```suggestion
let vscode = acquireVsCodeApi();
```

Reply:
It doesn't have to be declared again. I will use `const` as it is. Thank you!

Reply:
If `acquireVsCodeApi()` is called twice, it would cause an error.
The solution is creating a file like `media/CfgEditor/vscodeapi.js` and importing it :)